### DISCUSSION TITLE
The 3rd party plugins wiki page lost in transit

### DISCUSSION THREAD

**Discussion by tuffnatty** [pawelsalawa/sqlitestudio#3675] [Q&A]
### Details
It seems that the old wiki had a "3rd party plugins" page which got lost in transition to GitHub.

**Comment by pawelsalawa** [comment]
Maybe, although I don't remember if it really existed previously. Nevertheless I'm not aware of any 3rd party plugins to be existing at the moment. Do you?

**Comment by tuffnatty** [comment]
Ah, so you're also unaware ;-). There is a broken "3rd party plugins" link from the main wiki page, that's why I asked.
